docker-swarm/.env

```diff
@@ -0,0 +1 @@
+COMPOSE_PROJECT_NAME=autopilotmysql
```

Comment:
Brand new in 1.7.0 :innocent: (https://github.com/docker/compose/releases/tag/1.7.0)